src/dropdown/dropdown.component.ts

```diff
@@ -28,7 +28,9 @@ import { I18n } from "./../i18n/i18n.module";
 @Component({
 	selector: "ibm-dropdown",
 	template: `
-	<div class="bx--list-box">
+	<div
+		class="bx--list-box"
+		[ngClass]="{'bx--dropdown--light': theme === 'light'}">
```

Comment:
`@hostbinding` should be used instead to set the classes

Reply:
This class is not set on the host so it can't be set using host binding.